test_runner/src/test/kotlin/ftl/cli/firebase/test/networkprofiles/NetworkProfilesListCommandTest.kt

```diff
@@ -33,19 +32,18 @@ class NetworkProfilesListCommandTest {
 
     @Test
     fun run() {
-        val configurationsMock = emptyList<NetworkConfiguration>()
-        val prettyPrintSpy = spyk(prettyPrint)
-        every { prettyPrint } returns prettyPrintSpy
-        every {
-            any<Testing.TestEnvironmentCatalog.Get>().executeWithRetry()
-        } returns TestEnvironmentCatalog().apply {
-            networkConfigurationCatalog = NetworkConfigurationCatalog().apply {
-                configurations = configurationsMock
-            }
+        mockkObject(GcTesting) {
+            val configurationsMock = listOf(NetworkConfiguration().apply {
+                id = "test"
+                upRule = TrafficRule()
+                downRule = TrafficRule()
+            })
+            every {
+                GcTesting.networkConfiguration()
+            } returns configurationsMock
+            every { configurationsMock.asPrintableTable() } returns ""
+            CommandLine(NetworkProfilesListCommand()).execute()
+            verify { configurationsMock.asPrintableTable() }
```

Comment:
I think that we have response for this call-in mocked server. Could we use it somehow?

Reply:
Yes, we have calls to TestEnvironmentCatalog I change the test to using a mock server. Thanks!

Reply:
I'm not 100% sure but probably I need to mock it to check printing is executed on command run.